python/private/pypi/labels.bzl

```diff
@@ -14,6 +14,7 @@
 
 """Constants used by parts of pip_repository for naming libraries and wheels."""
 
+EXTRACTED_WHEEL_FILES = "extracted_whl_files"
```

Comment:
You still need to modify the `pkg_aliases` macro to include the extra alias.

Reply:
Done.